app/jobs/good_job_statsd_job.rb

```diff
@@ -16,30 +16,26 @@ def perform
     state_counts = filter.states
     gauge "count", state_counts
 
-    if ld_variation(key: "good_job.GoodJobStatsDJob.measure_staleness", default: true)
```

Comment:
I believe I had turned this off in LD

Reply:
I've [opened a PR](https://github.com/rubygems/rubygems.org/pull/5782) to reproduce the configured behavior. Please feel free to ship it if this is information we need sooner than later.